build/common.mk

```diff
@@ -100,18 +100,17 @@ ifeq ($(PULUMI_ROOT),)
 	PULUMI_ROOT:=$(shell realpath "$$HOME/.pulumi-dev")
 endif
 
-# Use Python 3 explicitly vs expecting that `python` will resolve to a python 3
-# runtime.
-PYTHON ?= python3
-PIP ?= pip3
+# Expecting that `python`, `pip` will resolve to a python 3 runtime.
+PYTHON ?= python
```

Comment:
Recent generations of Ubuntu ship python3 by default in `python` position. Windows workers do not have `python3`. Seems like a reasonable new default? But if we are not sure I can replace this with checking and auto-detection.

Reply:
AFAIK OS X is still shipping Python 2.7 as `python`, so I think we need to handle that case.

Reply:
Ah, true, interesting that the tests worked on macos-latest worker, perhaps that is because we reinstall Python on the worker. As for my local tests, I seem to be isolated from this as I install Python with Nix. I can handle this case.